src/scenes/AcceleratorRouter/ParticipantProjects/Reports/ReportsSettings.tsx

```diff
@@ -44,6 +60,12 @@ export default function ReportsSettings(): JSX.Element {
     }
   }, [specificMetricsResponse]);
 
+  useEffect(() => {
+    if (standardMetricsResponse && standardMetricsResponse.status === 'success') {
```

Comment:
This could be simplified to:

```suggestion
    if (standardMetricsResponse?.status === 'success') {
```